src/component/updateLegacy/index.js

```diff
@@ -859,29 +849,30 @@ updateLegacy.get = () => {
       data.state.header.area.align = 'center';
       return data;
     },
-    '4.41.0': function(data) {
+    '4.41.0': function (data) {
       data.state.header.search.newTab = false;
       return data;
     },
-    '4.42.0': function(data) {
+    '4.42.0': function (data) {
       data.state.group.openAll = {
         show: true,
         size: 1,
         style: 'box'
       };
       return data;
     },
-    '4.44.0': function(data) {
-      if (!'newTab' in data.state.link.item && 'newTab' in data.state.link) {
+    '4.44.0': function (data) {
+      // FIXME Eslint warned about !'newTab' (which is always "false in item && ..." so this block is never run)
+      if (!'newTab' in data.state.link.item && 'newTab' in data.state.link) { // eslint-disable-line
```

Comment:
This is a strange one. This update needs to check if `newTab` does not exist in this object. Not sure how else to perform the check here.

Reply:
Ah, yea that seems to resolve it.

Reply:
Yep, that fixed it